client/homebrew/editor/metadataEditor/metadataEditor.jsx

```diff
@@ -170,7 +160,9 @@ const MetadataEditor = createClass({
 			</div>
 			*/}
 
-			{this.renderAuthors()}
+			<StringArrayEditor label='authors'
```

Comment:
It is possible to delete the current users' name from the list. It comes back on page refresh, but perhaps we can add some property to StringArrayEditor to signal which strings should not be editable so the user's own name doesn't have the `x` button?

Reply:
That's a good idea -- need to figure out how to get the username into this element though

Reply:
Yeah the issue is that I looked through the element tree and haven't quite figured out where the username is and isn't available yet